### DISCUSSION TITLE
multiple windows

### DISCUSSION THREAD

**Discussion by joeboy8** [smplayer-dev/smplayer#366] [General]
When I originally installed smplayer, I could play more than one video file at a time (a second window would open when I double clicked the second file. This was by default. 
I had to uninstall and reinstall in Windows 11. Now, when I double click the second file, the original video ends and the second starts (no multiple windows).
Any idea why thinks changed and can I change it back? I thought I found the answer in Interface but not quite.

**Comment by smplayer-dev** [comment]
Disable the option "use only one running instance" in preferences -> interface -> instances.

**Reply by joeboy8** [reply]
As mentioned, I tried that, and I got a third window.

**Reply by joeboy8** [reply]
And I didn't have the third window on the original installation.
Thanks.

**Reply by joeboy8** [reply]
Good to go, but not like past version. Thanks. 
I currently have the toolbar and the video itself separate from the toolbar.  Is there a way to have the toolbar and the video connected as one window? I'll try not to bother you again.

**Reply by smplayer-dev** [reply]
That's weird. Go to preferences -> general -> video and be sure the video output is set to "default", "gpu" or "direct3d".

**Comment by joeboy8** [comment]
Ricardo,
Went to Options to get Preferences. Then, already set on general, no output visible. Clicked on Video tab, output driver was set on default. Tried gpu and then direct3d. Opened a previously played file on each and nothing changed...toolbar and actual video are separate windows.Smplayer Version 21.10.0 revision(10000)(64 bit). Don't see how Windows 11 Pro 64 bit could be making the difference, but I'm not a programmer.
Joe
P.S. Interesting not...if you minimize the video window, it doesn't pause. Minimize the tool bar window, video pauses. Makes sense, I guess.    On Saturday, February 5, 2022, 09:44:40 AM EST, Ricardo ***@***.***> wrote:  
 
 


That's weird. Go to preferences -> general -> video and be sure the video output is set to "default", "gpu" or "direct3d".

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by smplayer-dev** [reply]
Could you send the mpv log (menu View) after playing a video?

**Comment by joeboy8** [comment]
Here ya go (I hope)
    On Saturday, February 5, 2022, 10:37:00 AM EST, Ricardo ***@***.***> wrote:  
 
 


Could you send the mpv log (menu View) after playing a video?

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by smplayer-dev** [reply]
I'm afraid the log file is missing.

**Comment by joeboy8** [comment]
Ricardo,
The player ends when the files has played.
I need a step by step to get you what you want.
Joe
    On Saturday, February 5, 2022, 05:54:54 PM EST, Ricardo ***@***.***> wrote:  
 
 


I'm afraid the log file is missing.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by smplayer-dev** [comment]
I guess you enabled the option to close the player after playback. Anyway you can get the log during playback.

**Comment by joeboy8** [comment]
Here's Mplayer/mpv log
    On Sunday, February 6, 2022, 03:48:49 AM EST, Ricardo ***@***.***> wrote:  
 
 


I guess you enable the option to close the player after playback. Anyway you can get the log during playback.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by smplayer-dev** [reply]
Sorry, but the log didn't get here.

**Comment by joeboy8** [comment]
If you can't give me a step by step, I'll just try VLC or another player. What I attached looked like a log to me.

    On Sunday, February 6, 2022, 12:32:11 PM EST, Ricardo ***@***.***> wrote:  
 
 


Sorry, but the log didn't get here.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by smplayer-dev** [reply]
It seems you're replying to an email, and I think your attachment gets lost and doesn't arrive to github.
Anyway be sure the option "run mplayer/mpv in its own window", in preferences -> advanced -> mplayer/mpv is not enabled.
